Fedora 13 switched the default behaviour of the linker to no longer
indirectly link to required libraries (i.e. dependencies of a library
already linked to). Therefore we need to explicitly pass the depended on
libraries into the linker for building to work on Fedora 13.

More information is available on the Fedora Wiki:
https://fedoraproject.org/wiki/UnderstandingDSOLinkChange

JL - 15/06/10
Index: qemu-0.13.0/Makefile.target
===================================================================
--- qemu-0.13.0.orig/Makefile.target	2011-01-15 17:30:45.000000000 +0800
+++ qemu-0.13.0/Makefile.target	2011-01-15 17:33:22.000000000 +0800
@@ -193,7 +193,7 @@
 obj-y += rwhandler.o
 obj-$(CONFIG_KVM) += kvm.o kvm-all.o
 obj-$(CONFIG_NO_KVM) += kvm-stub.o
-LIBS+=-lz
+LIBS+=-lz -lX11 -ldl
 
 QEMU_CFLAGS += $(VNC_TLS_CFLAGS)
 QEMU_CFLAGS += $(VNC_SASL_CFLAGS)
